src/app/app.component.ts

```diff
@@ -202,12 +219,13 @@ export class AppComponent implements AfterViewInit {
   }
 
   async onLoginCompletedAsync() {
+    this.ref.detectChanges();
```

Comment:
can remove this line
- test that the login text is still updated after removal, but I think the state comparison fixed this instead